### DISCUSSION TITLE
Some questions about --minisat-dump-dimacs.

### DISCUSSION THREAD

**Discussion by TraceShadow** [cvc5/cvc5#10548] [Q&A]
Hi,
I'm trying to translate the smt2 files to dimacs, so I use `--minisat-dump-dimacs`.
Then I found that even I give different smt2 files to cvc5, it generates the same dimacs to me.
I'm sure the smt2 files are different, even though they only have very small differences.
Is that a bug or I do something wrong?
The smt2 files are shown as follow.
https://paste.ubuntu.com/p/9xPhZhsTGf/
https://paste.ubuntu.com/p/xTqrn5PThd/

**Comment by daniel-larraz** [comment]
Answered in #10551
